static/src/javascripts/projects/common/modules/ui/welcomeBanner.js

```diff
@@ -0,0 +1,48 @@
+define([
```

Comment:
I personally don't mind, but I think the convention is to have the filename as `welcome-banner`